maplibre_gl_example/lib/animate_camera.dart

```diff
@@ -73,6 +75,28 @@ class AnimateCameraState extends State<AnimateCamera> {
                   },
                   child: const Text('newCameraPosition'),
                 ),
+                if (!kIsWeb)
+                  TextButton(
+                    onPressed: () {
+                      mapController
+                          .easeCamera(
+                            CameraUpdate.newCameraPosition(
+                              const CameraPosition(
+                                bearing: 270.0,
+                                target: LatLng(46.233487, 14.363610),
+                                tilt: 30.0,
+                                zoom: 17.0,
+                              ),
+                            ),
+                            duration: const Duration(milliseconds: 500),
```

Comment:
Change lost among applied suggestions
```suggestion
                            duration: const Duration(seconds: 2),
```